### PR TITLE
Upgrade Pundit to v2

### PR DESCRIPTION
**Pull request by jmhooper**
**Why**: Because v2 of the gem has a `policy_class` option on `#authorize`. This is going to be helpful for me when I'm working on bulk adding users and authorizing that action.

### CODE REVIEW

**Comment by jmhooper** [comment]
Total coverage drop? Seriously?

**Comment by jmhooper** [comment]
Trying a rebase against master to see if that will fix the code coverage drop

File: Gemfile.lock [hunk 1]
```diff
@@ -290,7 +290,7 @@ GEM
     public_suffix (4.0.3)
     puma (4.3.3)
       nio4r (~> 2.0)
-    pundit (1.1.0)
+    pundit (2.1.0)
```
Comment (zachmargolis) [inline Gemfile.lock:293]: if we want >2 for a feature should we be explicit about that in `Gemfile` too?

Comment (jmhooper) [reply Gemfile.lock:293]: Doesn't hurt

**Pull request merged**